instructure/canvas-ios
# Use Firebase Remote Config for feature flags

This commit also separates experimental features to
be able to be turned on and off individually

affects: student, teacher, parent
refs: MBL-13547, MBL-13560
release note: none

Test Plan:
 - In student app turn on the new page details
   feature
 - Go to a group and open a page
 - You should be able to edit the page
 - Close the student app
 - Open it again (this close and open cycle may need to be
   done more than once)
 - Go into the experimental features screen from the developer
   menu
 - The testing experimental feature flag should be turned on
 - This is also the case in teacher and parent (may need to close it
   and open it a few times)
